karma-base.js

```diff
@@ -173,7 +173,7 @@ var notes_middleware = function (config) {
         });
       } else if (request.method === 'GET') {
         var src = path.resolve(image_path, query.name + '.png');
-        var img = new Buffer(fs.readFileSync(src)).toString('base64');
+        var img = Buffer.from(fs.readFileSync(src)).toString('base64');
```

Comment:
I am curious - can you provide some more information on new vs Buffer.from

Reply:
https://github.com/nodejs/node/wiki/Breaking-changes-between-v5-and-v6

Reply:
thanks